### ISSUE TITLE
Deep Learning COVID-19 Features on CXR using Limited Training Data Sets

### ISSUE THREAD

**Issue by HYUNMIN-HWANG** [yskim0/One-Day-One-Paper#45]
# Deep Learning COVID-19 Features on CXR using Limited Training Data Sets
- https://ieeexplore.ieee.org/document/9090149
- IEEE Transactions on Medical Imaging ( Volume: 39, Issue: 8, Aug. 2020)

---

## Summary
COVID-19 진단할 수 있으면서 상대적으로 적은 parameter수로 구성된 patch-based convolutional neural network 모델을 제안했다.
(we propose a patch-based convolutional neural network approach with a relatively small number of trainable parameters for COVID-19 diagnosis.)

![image](https://user-images.githubusercontent.com/70581043/128879995-13d0cc85-53ae-4378-8e3a-978586b48906.png)
과정은 크게 3 부분으로 나뉜다.
1. segmentation network (lung areas can be extracted)
2. decision is made based on the majority voting
3. probabilistic Grad-CAM saliency map is calculated to provide an interpretable result.

###  1. segmentation network
- chest radiography images로 부터 폐와 심장 가장자리를 추출하는 작업
- (FC)-DenseNet103
- cross entropy loss of multi-categorical semantic segmentation
- softmax probability

### 2. Classification network
- ResNet-18
- four classes: normal, bacterial pneumonia, tuberculosis (TB), and viral pneumonia( COVID-19 infection + SARS-cov or MERS-cov)
- implemented in two different versions
1. global approach (전체 그림을 한 번에 mask 하는 듯)
> - the masked images were resized to 224x224
> - global appearance of the CXR data
2. local approach (전체 그림을 랜덤하게 K번 mask 하는 듯) -> 해당 논문에서 제안하고자 하는 바
> - our proposed method
> - the masked images were cropped randomly with a size of 224x224

### 3. Grad-CAM saliency map visualization
- gradient weighted class activation map (Grad-CAM) _⛳ Grad-CAM이 뭐지??_
- to obtain the global saliency map, patch-wise Grad-CAM saliency maps should be weighted with the estimated probability of the class, and their average value should be computed

## RESULTS
1. segmentation
![image](https://user-images.githubusercontent.com/70581043/128881205-c0f46f3f-78fd-4a7d-9a0c-69ea412c8851.png)
- Jaccard similarity coefficient로 segmentaion 평가함
- 해당 논문에서 제안한 대로 전처리 했을 때 가장 높은 acc를 보였다.
![image](https://user-images.githubusercontent.com/70581043/128881402-633d23c0-4398-47ef-944b-476591e1913a.png)
segmentation 한 결과는 위 사진과 같다.

2. COVID-19 bio-markers
COVID-19 진단하는 결정적인 특징 : intensity distribution pattern within the lung area may be most effective in the diagnosis

3. Classification
![image](https://user-images.githubusercontent.com/70581043/128882031-7a627e3c-2394-4ec9-b8e1-f2c89891140c.png)
논문 모델 sensitivity of 92.5% for COVID-19 and viruses

4. Grad-CAM
![image](https://user-images.githubusercontent.com/70581043/128882154-02ab7751-257c-4253-8384-2e881ed8c971.png)
global approach로 분석하면 감염된 특정 부위가 아닌 넓은 부위로 인식한다. 반면, our local patch-based approach is more intuitive and interpretable compared to the existing methods.
